lib/declare_schema/model/habtm_model_shim.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+module DeclareSchema
+  module Model
+    class HabtmModelShim
```

Comment:
This class was wedged over in the migrator. It was time to move it out, and write a spec for it.